src/lib/precondition.ts

```diff
@@ -253,6 +279,42 @@ function getVariable<K extends LongKey, U extends FlatPreconditionValue[K]>(
   });
 }
 
+const slotMs = 3 * 60 * 1000; // 3 minutes
```

Comment:
We should add slot time to a graphql endpoint too since this will change in the future.